lib/swig/README.md

```diff
@@ -0,0 +1,52 @@
+#SWIG files
```

Comment:
#1191 is not related to SWIG . @stdevHan implement this file here and implement the feature in #1641 for #1568 .

Remove all files in `./lib/swig`